rocky/rocky/templates/tasks/boefjes.html

```diff
@@ -37,17 +37,21 @@ <h1>{% translate "Boefjes" %}</h1>
                                         <td>
                                             <a href="{% url 'plugin_detail' organization_code=organization.code plugin_type='boefje' plugin_id=task.p_item.data.boefje.id %}">{{ task.p_item.data.boefje.id }}</a>
                                         </td>
-                                        {% if task.status.value == "completed" %}
-                                            <td class="confirmation">
-                                                <span class="icon ti-check"></span>{{ task.status.value|capfirst }}
-                                            </td>
-                                        {% elif task.status.value == "failed" %}
-                                            <td class="warning">
-                                                <span class="icon ti-circle-x"></span>{{ task.status.value|capfirst }}
-                                            </td>
-                                        {% else %}
-                                            <td>{{ task.status.value|capfirst }}</td>
-                                        {% endif %}
+                                        <td class="nowrap">
+                                            {% if task.status.value == "pending" or task.status.value == "queued" %}
+                                                <i class="icon queued"></i>&nbsp;{{ task.status.value|capfirst }}
+                                            {% elif task.status.value == "running" or task.status.value == "dispatched" %}
+                                                <i class="icon running"></i>&nbsp;{{ task.status.value|capfirst }}
+                                            {% elif task.status.value == "cancelled" %}
+                                                <i class="icon cancelled"></i>&nbsp;{{ task.status.value|capfirst }}
+                                            {% elif task.status.value == "failed" %}
+                                                <i class="icon negative"></i>&nbsp;{{ task.status.value|capfirst }}
+                                            {% elif task.status.value == "completed" %}
+                                                <i class="icon positive"></i>&nbsp;{{ task.status.value|capfirst }}
+                                            {% else %}
+                                                <i class="icon neutral"></i>&nbsp;{{ task.status.value|capfirst }}
+                                            {% endif %}
```

Comment:
Not really blocking, but there seems to be a lot of duplicated between these templates that we should maybe refactor in another ticket?

Reply:
I think you're right. Could you create a ticket with the changes you want to see happening?